Snowdog_Components/components/Organisms/slider/_slider.scss

```
@@ -173,7 +173,7 @@
     }
 
     &--full-width {
-        width: 100vw;
+        width: calc(100vw - #{$scrollbar__size});
```

I did some testing and it looks like now width is a bit smaller than full width. When taking half scrollbar size it's perfect:

```suggestion
        width: calc(100vw - #{$scrollbar__size} / 2);
```